Sources_v3/StreamChatUI/AppearanceSetting.swift

```diff
@@ -30,18 +30,27 @@ public extension AppearanceSetting {
 /// An object describing the default appearance of the view. Be aware that the appearance object is generic-type-specific,
 /// in other words: `MyView<A>.defaultAppearance != MyView<V>.defaultAppearance`.
 private func fetchDefaultAppearance<T: AppearanceSetting>(_ key: String) -> Appearance<T> {
-    if let existing = _AppearanceStorage.shared.appearances[key] as? Appearance<T> {
+    if let existing = _AppearanceStorage.shared.appearance(for: key) as? Appearance<T> {
         return existing
     } else {
         let appearance = Appearance<T>()
-        _AppearanceStorage.shared.appearances[key] = appearance
+        _AppearanceStorage.shared.setAppearance(appearance, for: key)
         return appearance
     }
 }
 
 private class _AppearanceStorage {
     static let shared = _AppearanceStorage()
-    @Atomic var appearances: [String: Any] = [:]
```

Comment:
All appearance code is UIKit related. It means it should run on Main thread.
In this case whole `@Atomic` is redundant and instead we should have some kind of `AssertMainThread`.
Am I missing something?

Reply:
Yeah, that probably makes sense, why not. This is UI SDK anyway so we can do it in another PR